README

```diff
@@ -27,6 +27,10 @@ The basic features of Geany are:
 - embedded terminal emulation
 - extensibility through plugins
 
+Changelog
+---------
+See the release notes on `the website <https://www.geany.org/documentation/releasenotes/>`_ or `on GitHub <https://github.com/geany/geany/releases>`_.
+
```

Comment:
I wonder if we could make this change serve double-duty, something like:

```rst
Getting Geany
-------------------

The current version of Geany is 1.37 (see `Release Notes here <https://www.geany.org/documentation/releasenotes>`_). Official releases can be found on the `Releases Page <https://www.geany.org/download/releases>`_. Geany is also provided by a number of `Third-Party Packages <https://www.geany.org/download/third-party>`_.

See below for instructions on installing Geany from various release types.
```

Reply:
Yeah, except I'd leave the "here" out so "Release Notes" matches "Release Page" following.

Reply:
I like this more, also because it doesn't mix ChangeLog and ReleaseNotes which is not the same.
If we could omit the version number, the ReadMe doesn't need to be touched on each release.